spring-kafka/src/main/java/org/springframework/kafka/listener/KafkaMessageListenerContainer.java

```diff
@@ -83,7 +86,20 @@
 	 * @param topicPartitions the topics/partitions; duplicates are eliminated.
 	 */
 	public KafkaMessageListenerContainer(ConsumerFactory<K, V> consumerFactory, TopicPartition... topicPartitions) {
-		this(consumerFactory, null, null, topicPartitions);
+		this(consumerFactory, null, null, null, topicPartitions);
+	}
+
+	/**
+	 * Construct an instance with the supplied configuration properties, {@link ConsumerRebalanceListener} implementation
+	 * and specific topics/partitions - when using this constructor, {@link #setRecentOffset(long)
+	 * recentOffset} can be specified.
+	 * @param consumerFactory the consumer factory.
+	 * @param consumerRebalanceListener {@link ConsumerRebalanceListener} implementation.
+	 * @param topicPartitions the topics/partitions; duplicates are eliminated.
+	 */
+	public KafkaMessageListenerContainer(ConsumerFactory<K, V> consumerFactory, ConsumerRebalanceListener consumerRebalanceListener,
```

Comment:
We would like to avoid the code line length more than 120 symbols: https://github.com/spring-projects/spring-integration/wiki/Spring-Integration-Framework-Code-Style.
Right, this is minor and can be fixed on merge.
Just want to share with you our code style rules with the hope that you may follow them in the future contribution to let us avoid extra work during review/merge.